bodhi/server/mail.py

```diff
@@ -281,10 +281,40 @@
 
 --------------------------------------------------------------------------------
 %(notes)s%(changelog)s%(references)s
+This update can be installed with the "dnf" update programs.  Use
+su -c 'dnf%(yum_repository)s update %(name)s' at the command line.
+For more information, refer to "Package Management",
+available at https://docs.fedoraproject.org
+
+All packages are signed with the Fedora EPEL GPG key. More details on the
+GPG keys used by the Fedora Project can be found at
+https://fedoraproject.org/keys
+--------------------------------------------------------------------------------
+"""
+
+# message template for legacy systems, still pointing users to use "yum"
+fedora_epel_legacy_errata_template = u"""\
+--------------------------------------------------------------------------------
+Fedora EPEL%(testing)s Update Notification
+%(updateid)s
+%(date)s
+--------------------------------------------------------------------------------
+
+Name        : %(name)s
+Product     : %(product)s
+Version     : %(version)s
+Release     : %(release)s
+URL         : %(url)s
+Summary     : %(summary)s
+Description :
+%(description)s
+
+--------------------------------------------------------------------------------
+%(notes)s%(changelog)s%(references)s
 This update can be installed with the "yum" update programs.  Use
 su -c 'yum%(yum_repository)s update %(name)s' at the command line.
-For more information, refer to "Managing Software with yum",
-available at https://docs.fedoraproject.org/yum/.
+For more information, refer to "Package Management",
+available at https://docs.fedoraproject.org
```

Comment:
We should probably link to documentation about Red Hat Enterprise Linux here, since that URL isn't about enterprise linux and yum isn't covered in the recent versions of dnf documentation there. I found this link with duckduckgo.com but it doesn't seem to respond right now:

https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Deployment_Guide/ch-yum.html

Presumably there is one for Red Hat Enterprise Linux 7 as well.